README.md

```diff
@@ -1,2 +1,50 @@
-# docker-redis
-Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache, and message broker
+# Docker Redis
+
```

Comment:
Thought of adding a badge here for the latest Docker image on Docker Hub but felt like I was getting distracted. Maybe later.